website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -545,13 +555,35 @@ var QuickSearchNodeDisplay = {
             return m('.', args.eligibleNodes().slice(0, args.countDisplayed()).map(function(n){
                 var project = args.nodes()[n];
                 var numContributors = project.embeds.contributors.links.meta.total;
+                var title = project.attributes.title;
+                var root;
+                try {
+                    root = project.embeds.root.data.attributes.title;
```

Comment:
Try to keep `try` blocks as constrained as possible to avoid catching an error that isn't meant to be caught and mishandling it. 

Also, as mentioned in a previous comment, you can use `lodash.get` to get deeply nested attributes.